draft-ietf-quic-tls.md

```diff
@@ -1279,8 +1279,7 @@ An application-layer protocol MAY restrict the QUIC versions that it can operate
 over.  Servers MUST select an application protocol compatible with the QUIC
```

Comment:
Plural singular mix: Servers vs a client

Reply:
Not part of this PR.